<h2> Bioinformatics Translation Consensus Sequences </h2>
  
<p>This repository contains a .zip file of appendices containing all files and programs used to conduct an collaborative exploration of Yeast 
  Translation Consensus Sequences for my bioinformatics final project (Spring 2022), an appendices and sources document, the entire project write up, 
  and the project directions file .</p>
  
<p><strong>Abstract: </strong></p>
<p>
Molecular biology has greatly benefited from the use of information theory (IT), which was originally developed for the analysis of communication 
  systems, but has been useful in genome global analysis, classification of motifs, and specifically to this exploration: the prediction of 
  transcription factor binding sites and sequence characterization based on local sequences. Information theory is used to analyze different 
  parts and features of the sequences surrounding the annotated (known) start site of yeast genes from the dev_Yest350 database, and focused on 
  examining aspects of sequences surrounding the start site of genes in the top 10% of protein expression, the middle 10%, from 45% to 55%, of 
  protein expression, and the bottom 10% of protein expression. A general exploration of information content was conducted on sequences 10 nucleotides 
  before (-10) the known start sequence (annAUG) and 10 nucleotides after the annAUG (+10). In addition, further investigations of these data were 
  conducted, which includes using information content to predict consensus sequences of nucleotides -6 to -1 relative to the annAUG, comparing the 
  frequency of G content to the frequency of GNN codons, and determining and comparing information content of sequences with upstream ATGs in the 
  5â€™ UTR and sequences without upstream ATGs.
</p>

<p><strong>Appendices and Sources: </strong></p>
<p>Descriptions of all files and appendices can be found in the Appendices and Sources documents, which provide in depth explanations on purpose of each
file of code, function, and text files used. </p>
